crates/uv-python/src/managed.rs

```diff
@@ -362,13 +360,14 @@ impl ManagedPythonInstallation {
             }
             // PyPy uses a full version number, even on Windows.
             ImplementationName::PyPy => format!("{}.{}", self.key.major, self.key.minor),
-            ImplementationName::GraalPy => {
-                unreachable!("Managed installations of GraalPy are not supported")
-            }
+            ImplementationName::GraalPy => String::new(),
```

Comment:
empty string?

Reply:
Yes, GraalPy does not encode a version in the executable.